x-pack/test/case_api_integration/common/lib/authentication/index.ts

```diff
@@ -61,7 +73,8 @@ export const deleteSpaces = async (getService: CommonFtrProviderContext['getServ
     }
   }
 };
-const deleteUsersAndRoles = async (getService: CommonFtrProviderContext['getService']) => {
+
+export const deleteUsersAndRoles = async (getService: CommonFtrProviderContext['getService']) => {
```

Comment:
Exporting this so the `security_only` tests can call it directly and not create any spaces.